src/Entity/Contact.php

```diff
@@ -112,6 +113,23 @@ public function isValidStatus(string $status): bool
         ], true);
     }
 
+    public function getTags(): array
+    {
+        return $this->tags;
+    }
+
+    public function setTags(array $tags): self
+    {
+        $this->tags = $tags;
+        return $this;
+    }
+
+    public function addTag(string $key, string $value): self
+    {
+        $this->tags[$key] = $value;
```

Comment:
As written in the official [documentation](https://emailoctopus.com/api-documentation/lists/create-contact) this should be a simple array of strings and not a key => value array.
Could you please change it and add a check if the tag already exists?

Reply:
Hi, good day.
The key/value pair will work if the value is only true/false.

The updates is working, and this is the result:
![EmailOctopus-Result](https://github.com/Ideneal/EmailOctopus/assets/38401585/4543bfdc-5db2-4097-92e3-ff0349edc6b4)

Reply:
Sorry, I don't understand....why the value should be true or false if it's a string?